ScanCommand/src/main/java/io/ballerina/scan/internal/ScanCmd.java

```diff
@@ -150,6 +150,15 @@ public void execute() {
             outputStream.println();
             outputStream.println("View scan results at:");
             outputStream.println("\t" + reportPath + System.lineSeparator());
+
+            if (scanReport) {
+                Path scanReportPath;
+                outputStream.println("Generating scan report...");
+                scanReportPath = ScanUtils.generateScanReport(issues, project.get(), targetDir);
+                outputStream.println();
+                outputStream.println("View scan report at:");
+                outputStream.println("\t" + ScanToolConstants.FILE_PROTOCOL + scanReportPath + System.lineSeparator());
```

Comment:
Can use multiline strings like,

```suggestion
                outputStream.println("""
                      View scan report at:
                          ${ScanToolConstants.FILE_PROTOCOL + scanReportPath}
                      """);
```

Reply:
This does not seem to work with Java 17, will be using the previous approach.